src/shared/exec-util.h

```diff
@@ -14,12 +14,19 @@ enum {
         _STDOUT_CONSUME_MAX,
 };
 
+typedef enum {
+        EXECDIR_NONE          = 0,      /* No execdir flags */
+        EXECDIR_PARALLEL      = 1 << 0, /* Execute scripts in parallel, if possible */
+        EXECDIR_IGNORE_ERRORS = 1 << 1, /* Ignore non-zero exit status of scripts */
+} ExecDirFlags;
```

Comment:
nitpick: let's keep the CamelCase of the type and the UPPER_AND_UNDERSCORE formatting of the values somewhat in sync, i.e. EXECDIR_XYZ → EXEC_DIR_XYZ would be nicer I think

Reply:
Would certainly be more consistent - changed.